libs/reactive-security/src/main/java/no/nav/testnav/libs/reactivesecurity/domain/AzureNavProxyClientCredential.java

```
@@ -0,0 +1,41 @@
+package no.nav.testnav.libs.reactivesecurity.domain;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Objects;
+
+import no.nav.testnav.libs.securitycore.domain.azuread.ClientCredential;
+
+@Configuration
+public class AzureNavProxyClientCredential extends ClientCredential {
+    private final String tokenEndpoint;
+
+    public AzureNavProxyClientCredential(
+            @Value("${AZURE_NAV_OPENID_CONFIG_TOKEN_ENDPOINT:#{null}}") String tokenEndpoint,
```

#{null} betyr jo at oppstart ikke feiler selv AZURE_NAV_OPENID_CONFIG_TOKEN_ENDPOINT ikke er definert.
Selv tenker jeg at "fail fast" er en god strategi, spesielt i forhold til oppstart, da det kan lette feilsøkingen: en udefinert value gir klartekst feilmelding fra Spring.